green/test/test_result.py

```diff
@@ -104,6 +104,16 @@ def test_displayStderr(self):
 
 
 class TestProtoTestResult(unittest.TestCase):
+    def test_startStop(self):
+        """
+        startTest/stopTest work correctly
+        """
+        ptr = ProtoTestResult()
+        test = proto_test(MagicMock())
+        ptr.startTest(test)
+        ptr.stopTest(test)
+        self.assertGreater(float(ptr.test_time), 0)
```

Comment:
For readability it is better to use plain words:
```suggestion
        result = ProtoTestResult()
        test = proto_test(MagicMock())
        result.startTest(test)
        result.stopTest(test)
        self.assertGreater(float(result.test_time), 0)
```

That said I have a bit of concern about this because time is not mocked and on a future very fast machine it would not be impossible for the time to be recorded as a true zero.

We could add https://pypi.org/project/freezegun/ in our requirements-dev.txt file and use it to mock the time to ensure that we have predictable behavior.

Reply:
It's better to be consistent with other tests. This is the naming and the structure used in all other tests in TestProtoTestResult.

I don't think we need to worry about time being zero. Even calling `time.time()` in a tight loop does not produce the same number. If it does become a problem, we can simply add a small sleep. `freezegun` sets time to a constant, which will result in test_time being zero. If we need to mock, we can directly mock `time.time()` to an iterable, so it returns different values on subsequent calls.

@CleanCut what do you think?

Reply:
Added mocking for `time.time()` and a more precise assert.